src/type_manipulation.hpp

```diff
@@ -13,47 +13,59 @@ class TypeManipulation {
      * This function is purely here because the STD variant doesn't work. It's a literal copy of strcpy. dest = destination. src =
      * source.
      *
-     * @param dest : char *
-     * @param src : const char *
-     * @return char*
+     * @param Destination string.
+     * @param Source string.
+     * @return void
```

Comment:
parameters should mention the variable and then their meaning
return is not necessary if it is void, but it is now char again. See all other functions here too

```* @param[in]     a    What is the input for the function```
``` * @return What will the function return```